test/unit/kubernetes-deploy/kubernetes_resource/pod_test.rb

```diff
@@ -196,6 +196,7 @@ def build_synced_pod(template)
       logger: @logger, deploy_started_at: Time.now.utc)
     mediator = KubernetesDeploy::SyncMediator.new(namespace: 'test', context: 'minikube', logger: logger)
     mediator.expects(:get_instance).with('Pod', anything).returns(template)
+    KubernetesDeploy::ContainerLogs.any_instance.stubs(:sync)
```

Comment:
was this the test that was invoking `kubectl`?

Reply:
Yep! Every test using this helper (so most of them in this suite) were actually trying to sync logs.